src/fmain.pas

```diff
@@ -6723,14 +6813,42 @@ procedure Tfrmain.milistPropertiesClick(Sender: TObject);
         frnewdown.btnStart.Caption:=fstrings.btnpropertiesok;
         frnewdown.btnStart.GlyphShowMode:=gsmNever;
         frnewdown.cbDestination.Text:=fstrings.nochangefield;
-        frnewdown.deDestination.Text:=fstrings.nochangefield;
-        frnewdown.cbEngine.Items.Add(fstrings.nochangefield);
-        frnewdown.cbEngine.ItemIndex:=frnewdown.cbEngine.Items.IndexOf(fstrings.nochangefield);
-        frnewdown.edtParameters.Text:=fstrings.nochangefield;
-        frnewdown.edtPassword.Text:=fstrings.nochangefield;
-        frnewdown.edtUser.Text:=fstrings.nochangefield;
-        frnewdown.cbQueue.Items.Add(fstrings.nochangefield);
-        frnewdown.cbQueue.ItemIndex:=frnewdown.cbQueue.Items.IndexOf(fstrings.nochangefield);
+
+        if deDestinationTextIsSame = 1 then
+          frnewdown.deDestination.Text:=deDestinationText
```

Comment:
showing an exact value when all the selected items already have the very same exact value

Reply:
yes i agree